target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -63,6 +63,11 @@ ipq40xx_setup_interfaces()
 	engenius,eap2200)
 		ucidef_set_interface_lan "eth0 eth1"
 		;;
+	buffalo,wtr-m2133hp)
```

Comment:
Can we merge this case with "ezviz,cs-w3-wd1200g-eup"?

Reply:
When I merge this into "ezviz,cs-w3-wd1200g-eup", both eth0 and eth1 are assigned to LAN (VLAN ID 1) in Luci. If it is normal behavior or a Luci bug, it can be merged.
![01](https://user-images.githubusercontent.com/41878038/78419587-1b0ec400-7682-11ea-9291-5082a2f98a44.jpg)